src/components/auth/backup/BackupDevice.svelte

```diff
@@ -1,11 +1,26 @@
 <script lang="ts">
+  import clipboardCopy from 'clipboard-copy'
+  import QRCode from 'qrcode-svg'
   import { createEventDispatcher } from 'svelte'
 
+  import { sessionStore, theme } from '../../../stores'
   import type { BackupView } from '$lib/views'
   import ClipboardIcon from '$components/icons/ClipboardIcon.svelte'
 
   const dispatch = createEventDispatcher()
 
+  const origin = window.location.origin
+  const connectionLink = `${origin}/link?username=${$sessionStore.username}`
+  const qrcode = new QRCode({
+    content: connectionLink,
+    color: $theme === 'light' ? '#334155' : '#E2E8F0',
```

Comment:
Unfortunately, these colors live outside our theme. A couple of approaches for using the theme colors are mentioned here: https://github.com/saadeghi/daisyui/discussions/415#discussioncomment-1995833

We should discuss these options. Both are fraught in their own ways.

Reply:
Hmmm... importing DaisyUI theme didn't look too bad to me. Let's talk about it next week!